### ISSUE TITLE
Battery voltage list causes a crash when adding it to the builder

### ISSUE THREAD

**Issue by MichaelSwan3** [dronefleet/mavlink#59]
Hi,

I'm trying to create my own batterystatus messages for testing. However it crashes everytime I create a voltage list.

Also, I don't know if this is related, but when I connected to the simulated drone in mission planner, voltages list is always null.

Creating voltages in BatteryStatus
 ```
 public MavMsg<BatteryStatus> createBatteryStatus(int id, int voltage, int batteryRemaining, int currentBattery) {

        /*
        // add 10 cells
        List<Integer> voltages = new ArrayList<Integer>();
        voltages.add(voltage);
        voltages.add(65535);
        voltages.add(65535);
        voltages.add(65535);
        voltages.add(65535);
        voltages.add(65535);

        voltages.add(65535);
        voltages.add(65535);
        voltages.add(65535);
        voltages.add(65535);
*/

        BatteryStatus batteryStatus = BatteryStatus.builder()
                .batteryRemaining(batteryRemaining)
                .currentBattery(currentBattery)
                .id(id)
               // .voltages(voltages)

                .build();

        // crashes here if we try to add voltage
        MavMsg<BatteryStatus> mmbat = new MavMsg<BatteryStatus>(batteryStatus);
```

Mission planner battery setup
![image](https://user-images.githubusercontent.com/43261902/114653575-ae65e500-9d2b-11eb-83b2-49b5d3419620.png)

Voltages are always null
![image](https://user-images.githubusercontent.com/43261902/114653677-ea994580-9d2b-11eb-95a1-bfe8c93f0bc8.png)

It makes me think that there could be an issue with voltages in BatteryStatus? Most likely I've done something wrong though.

Cheers

**Comment by MichaelSwan3** [comment]
Specifically it crashes here:
![image](https://user-images.githubusercontent.com/43261902/114656343-3dc1c700-9d31-11eb-8448-bca77d270420.png)

![image](https://user-images.githubusercontent.com/43261902/114656383-50d49700-9d31-11eb-97e4-516926ff91e6.png)

**Comment by MichaelSwan3** [comment]
It crashes when it tries to convert it into bytes

![image](https://user-images.githubusercontent.com/43261902/114657227-f63c3a80-9d32-11eb-9f58-3e1c6bfc682e.png)

![image](https://user-images.githubusercontent.com/43261902/114657323-2a176000-9d33-11eb-82de-6c6df253b336.png)

So, it doesn't have a way of converting List<integer> into a set of bytes.

**Comment by MichaelSwan3** [comment]
It looks like it might be fixed in here.
https://github.com/dronefleet/mavlink/commit/2255e6df02974af2031c05dcb509309ce75bdb5e

We are using the version from here

> Get it
> Maven Central coordinates are io.dronefleet.mavlink:mavlink:1.1.8

Has this been updated with the latest changes and if not, is it possible to get it updated ?

**Comment by benbarkay** [comment]
This fix was indeed present in `master`.
I've released version 1.1.9. Should be available in central shortly.

**Comment by MichaelSwan3** [comment]
Ben,
I've got the new version and so far it's working great - 
Thanks!
